InstantSearch.js/routing-seo-friendly/src/search-router.js

```diff
@@ -0,0 +1,101 @@
+/* global instantsearch */
+
+function getCategorySlug(name) {
+  return name.replace(/\s/g, '+');
+}
+
+function getCategoryName(slug) {
+  return slug.replace(/\+/g, ' ');
+}
+
+const router = instantsearch.routers.history({
+  windowTitle({ category, query }) {
+    const queryTitle = query ? `Results for "${query}"` : 'Search';
+
+    if (category) {
+      return `${category} – ${queryTitle}`;
+    }
+
+    return queryTitle;
+  },
+
+  createURL({ qsModule, routeState, location }) {
+    let baseUrl = location.href.split('/search/')[0];
```

Comment:
I don't know how to reproduce it but I manage to have an URL with multiple times `/search` in it.

Reply:
https://yjojj4lnrz.codesandbox.io/search?query=apple&brands/search/?query=search&brands=

if the query is "search"

Reply:
(ignore this, I can't find how I got this url anymore) :'(

Reply:
The issue occurs when we omit the trailing slash inside the URL. Click on the URL below and trigger a refinement, the URL should not contain two times `/search`. The last occurence is the correct URL.  We could use a RegExp to avoid this issue.

> https://yjojj4lnrz.codesandbox.io/search?query=apple

Reply:
Fixed in 21788d37a6a9f6f1a90c72cd0cbe17bb389a28f3.